pkgs/applications/virtualization/docker/default.nix

```diff
@@ -48,6 +57,6 @@ stdenv.mkDerivation rec {
     description = "An open source project to pack, ship and run any application as a lightweight container";
     license = licenses.asl20;
     maintainers = with maintainers; [ offline tailhook ];
-    platforms = platforms.linux;
```

Comment:
You can use 'optional clientOnly platform.darwin' here instead, just as a note, don't need to change now.